python/cudf/cudf/tests/test_datetime.py

```diff
@@ -2563,3 +2563,17 @@ def test_date_range_start_end_divisible_by_freq():
     result = cudf.date_range("2011-01-01", "2011-01-02", freq="h")
     expected = pd.date_range("2011-01-01", "2011-01-02", freq="h")
     assert_eq(result, expected)
+
+
+def test_writable_numpy_array():
+    gi = cudf.Index([1, 2, 3], dtype="datetime64[ns]")
```

Comment:
Could we just do

```python
gi = cudf.Index(...).to_numpy()
pi = pd.Index(...).to_numpy()

assert gi.flags == pi.flags
```

instead?

Reply:
the `ownable` flag is not the same for both so we will not be able to this assertion.